frontend/src/ches/Performance.jsx

```diff
@@ -0,0 +1,37 @@
+import React,{Component} from 'react';
+
+
+class Performance extends Component {
+
+  render() {
+
+    return (
+      <div className="container" id="maincontainer">
+        <div id="mainrow" className="row">
+          <div className="col-md-12 mb-3">
+            <iframe id="metabaseDashboard" src={window._env_.REACT_APP_CHES_PERFORMANCE_URL} frameBorder="0" width="100%" scrolling="no" allowtransparency=""></iframe>
+          </div> 
+        </div>
+      </div>
+    );
+  }
+
+  // link to external js script that makes iframe responsive
+  componentDidMount() {
+    const script = document.createElement("script");
+    script.src = "https://metabase-domo-domo.pathfinder.gov.bc.ca/app/iframeResizer.js";
```

Comment:
I would concur with this - we should try to avoid depending on loading other external js resources whenever possible. I think it would be worthwhile in this case to static host that javascript in the public folder. Or, if there was a way to actually formally import this js file as part of our project and let webpack manage it even better.

Reply:
Yeah metabase-domo-domo.pathfinder.gov.bc.ca is linked from another project's Metabase instance (the one in DOMO), which is not guaranteed to be around/a matching version/accessible.

The 'standard' way to get this file from MB is to fetch it from the metabase instance that the iFrame is referencing, which in this case would definitely not be the DOMO one.

Or could, as Jason and Jeremy are saying, include it in the MSSC project itself. The advantage of pulling it rather than including it is that if MB itself updates the iFrameResizer script is going to be in snyc with the iframeResizer.contentWindow.min.js script in the remote iFrame itself... But I think it's fairly static...